prow/config/jobs.go

```diff
@@ -101,6 +101,8 @@ type Presubmit struct {
 	Spec *v1.PodSpec `json:"spec,omitempty"`
 	// Run these jobs after successfully running this one.
 	RunAfterSuccess []Presubmit `json:"run_after_success"`
+	// Defines if a test is a required check as defined in the Github Branch protection.
+	NonRequiredCheck bool `json:"non_required_check,omitempty"`
```

Comment:
Please avoid double negatives.

`if optional` is easier on the brain than `if !NonRequiredCheck`

Reply:
well I have to make nonrequired default so I don't have much choice.

Reply:
Choose a better name than non-required, look up [antonyms of required](http://www.thesaurus.com/browse/required) if you don't like my naming suggestion (optional).

`optional = false` is easier to understand than `nonrequired = false`

Reply:
you are right ! Yes optional is better, sorry somehow I understood that optional was reverse of non required. I ll change it.